package.json

```diff
@@ -42,6 +42,12 @@
     "@bower_components/crypto": "anodynos/node2web_crypto#3.46.1",
     "@bower_components/international-phone-number": "mareczek/international-phone-number#^0.0.16",
     "@bower_components/matchmedia": "paulirish/matchMedia.js#~0.2.0",
+    "@ovh-ux/angular-uirouter-title": "^1.0.4",
+    "@ovh-ux/manager-core": "^1.0.0-alpha.0",
+    "@ovh-ux/manager-freefax": "^1.0.0-alpha.1",
+    "@ovh-ux/manager-overthebox": "^1.0.0-alpha.1",
+    "@ovh-ux/manager-universes-styles": "^1.0.0-alpha.0",
+    "@ovh-ux/ovh-angular-contracts": "https://github.com/ovh-ux/ovh-angular-contracts.git#v3.x.x",
```

Comment:
Quick reminder that we have to release this dependency.

Reply:
Done using `^3.0.0-beta.0`